config/schemas/config.yaml

```diff
@@ -14,6 +14,291 @@ $defs:
     thing:
       $ref: "#/$defs/thing"
     ```
+  affinity:
+    $comment: Real k8s resources might have existing schemas that could be reused as-is
+    additionalProperties: false
+    default:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: node-role.kubernetes.io/control-plane
+                  operator: Exists
+    properties:
```

Comment:
I think `nodeAffinity` and `podAffinity` might need to be added here, as atleast for `nodeAffinity` the following schema validation failed:

```console
[ck8s] Validating wc config
[ck8s] Failed schema validation:
...
wc-config.yaml: ingressNginx.defaultBackend.affinity: Additional property nodeAffinity is not allowed
```